src/main/java/sirius/db/mixing/BaseEntity.java

```diff
@@ -66,12 +66,43 @@
     /**
      * Returns the descriptor which maps the entity to the database table.
      *
-     * @return the ddescriptor which is in charge of checking and mapping the entity to the database
+     * @return the descriptor which is in charge of checking and mapping the entity to the database
      */
     public EntityDescriptor getDescriptor() {
         return mixing.getDescriptor(getClass());
     }
 
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used as a consequence describing why an error happened.
+     * For instance, when an entity's required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
```

Comment:
I don't understand really understand this explanation :sweat_smile:  Maybe try to make it a bit clearer...

Reply:
Where, there is an example! Not sure how better I could describe it 🤔

Reply:
I mean which of the 2 variants is now actually logged?
It just says that "it would be nice to tell the entity", but do we always do that now, or are both variants still possible?